Sources/SnowManLogic/GameLogic/guessManager.swift

```diff
@@ -32,21 +32,20 @@ public class GuessManager {
 		return guess == word
 	}
 
-	public func appendWordPlaceholder(word: String, guess: String) {
-		let fullWordGuessPlaceholder = "#"
-		if (correctlyGuessedFullWord(word, guess: guess)) {
-			correctGuesses.append(fullWordGuessPlaceholder)
+	public func appendGuess(word: String, guess: String) {
+		if (isGuessingFullWord(guess)) {
+			addGuessToCollection(correctlyGuessedFullWord, word, guess)
 		} else {
-			incorrectGuesses.append(fullWordGuessPlaceholder)
+			addGuessToCollection(hasGuessInWord, word, guess)
```

Comment:
why are you passing this private method to another private method, rather than just calling it?